flink-cdc-dist/src/main/flink-cdc-bin/bin/flink-cdc.sh

```diff
@@ -34,11 +34,11 @@ if [[ -z $FLINK_HOME ]]; then
   exit 1
 fi
 
-# Setup Flink related configurations
-# Setting _FLINK_HOME_DETERMINED in order to avoid config.sh to overwrite it
-_FLINK_HOME_DETERMINED=1
```

Comment:
Should `_FLINK_HOME_DETERMINED` env var be preserved here?

Reply:
The variable _FLINK_HOME_DETERMINED is not used. Is this variable used somewhere in the code?

Reply:
I think it was used to tell Flink's `config.sh` not to override given `FLINK_HOME` variable, and should be set before calling `. $FLINK_HOME/bin/config.sh`.

Reply:
Ok, I got it.